migtests/tests/oracle/basic-live-test/init-db

```diff
@@ -7,9 +7,17 @@ source ${SCRIPTS}/functions.sh
 
 echo "Assigning permissions to the validate files"
 chmod +x ${TEST_DIR}/validate
+chmod +x ${TEST_DIR}/validateAfterChanges
 
-echo "Deleting existing data in target database"
+echo "Deleting existing data in source and FF database"
```

Comment:
I'd suggest separating out the init-db for source and ff. 
in live-migration test, we need not init ffdb